.github/workflows/tests.yml

```diff
@@ -5,6 +5,14 @@ on:
     branches: [master]
   pull_request:
     branches: [master]
+    types: 
+      - assigned
+      - opened
+      - edited
+      - syncrhonize
+      - reopened
+      - ready_for_review
+      - review_requested
```

Comment:
I would say we can get rid of `edited`, `assigned` and `review_requested`, as none of these imply that the underlying code has actually changed.